pytheus/metrics.py

```diff
@@ -153,6 +180,12 @@ def count_exceptions(
             self.inc()
             raise
 
+    def collect(self) -> Sample:
+        # TODO: probably need a way to add default metric creation labels
```

Comment:
what do you mean by "default metric creation labels"?

Reply:
the labels you will be able to define when you create a new metric for example: `create_counter('name', required_labels=['a', 'b'], default_labels={'a': '1'}`

We will need to find the best place to do the merging of Child metric labels with Parent default labels for the sample, but that also might depends on the Registry implementation